qa/suites/upgrade/octopus-x/parallel/1-ceph-install/octopus.yaml

```diff
@@ -0,0 +1,68 @@
+meta:
+- desc: |
+   install ceph/octopus latest
+   run workload and upgrade-sequence in parallel
+   upgrade the client node
+tasks:
+- install:
+    #exclude_packages:
+      #- ceph-mgr-cephadm
+      #- cephadm
+      #- libcephfs-dev
+    branch: octopus
+- print: "**** done installing octopus"
+- cephadm:
+    roleless: true
+    cephadm_branch: octopus
+    #use image from for example dockerhub
+    #image: docker.io/ceph/daemon-base:latest-octopus-devel
+    #from  https://quay.io/repository/ceph-ci/ceph
+    branch: octopus
+- print: "**** done cephadm: octopus, ceph and daemons in containers setup"
+- cephadm.shell:
+    host.a:
+      - ceph orch status
+      - ceph orch ps
+      - ceph orch ls
+      - ceph orch host ls
+      - ceph orch device ls
+- print: "**** done cephadm.shell"
+- cephadm:
```

Comment:
these settings should go in the first ```- cephadm:``` block, there's no need for this 2nd one - that's causing the current problem because this 2nd one expects a mgr in roles